src/commands/CustomCommands.ts

```diff
@@ -0,0 +1,17 @@
+import { CustomCommand } from '../types';
+
+export const customCommands: Array<CustomCommand> = [
+  {
+    command: "runelite",
+    message: "",
+    image: "https://imgur.com/tv0Jro8"
+  },
+  {
+    command: "verified",
+    message: "To verify your group, please DM any Mod/Admin with proof of ownership. This must be in the form of a screenshot with:\n\n- The clan chat tab open, showing your rank of Gold Star (or keyholder)\n- Your WOM group number, your discord name, and today’s date typed in the chatbox.\n\nAfter verification is complete, we can reset the verification code for you if needed"
+  },
```

Comment:
We will probably want an image here to give an example of what we expect. We also might be changing the requirements because of the custom ranking with the new clans system.